src/common/layers/Layer.js

```diff
@@ -86,15 +86,12 @@ export default class Layer extends Observable {
         writable: true,
       },
       copyright: {
-        get: () => {
-          return this.get('copyright', copyright);
-        },
-        set: (newCopyright) => {
-          return this.set('copyright', newCopyright);
-        },
+        value: copyright,
+        writable: true,
       },
       visible: {
         value: visible === undefined ? true : visible,
+        writable: true,
```

Comment:
of course not why do you think there is still a `setVisible`  function ?

Reply:
ignore the comment , i see the problem

Reply:
Yes readyOnly param will then not be possible to be reassigned by further components.